lib/signs_ui/alerts/alert.ex

```diff
@@ -0,0 +1,44 @@
+defmodule SignsUi.Alerts.Alert do
+  defstruct [:id, :created_at, :service_effect]
+
+  @type id :: String.t()
+
+  @type t :: %__MODULE__{
+          id: id(),
+          created_at: DateTime.t() | nil,
+          service_effect: String.t() | nil
+        }
+
+  defimpl Jason.Encoder, for: SignsUi.Alerts.Alert do
+    def encode(value, opts) do
+      Jason.Encode.map(
+        %{
+          id: value.id,
+          created_at: format_dt(value.created_at),
+          service_effect: value.service_effect
+        },
+        opts
+      )
+    end
+
+    @spec format_dt(DateTime.t() | nil) :: String.t()
+    defp format_dt(nil) do
+      ""
+    end
+
+    defp format_dt(dt) do
+      today = "America/New_York" |> DateTime.now!() |> DateTime.to_date()
+
+      date =
+        if DateTime.to_date(dt) == today do
+          "Today"
+        else
+          Calendar.strftime(dt, "%m/%d")
+        end
+
+      time = Calendar.strftime(dt, "%-I:%M %p")
+
+      "#{date} @ #{time}"
```

Comment:
This seems like we're embedding presentational logic in the backend data model. Can we emit this in the JSON as ISO8601, and have the React app format it? (as with e.g. `expiration_to_iso8601`)

Reply:
I was trying to avoid having to pull in `moment` or some other big JS library for time zone conversions (since "today" has to be time zone aware). But good catch about how we already are sending over `iso8601` - I forgot the browser already handles locales for us. Yeah, I'll make this switch. In fact, it means I don't need to prove my own `defimpl Jason.Encoder` anymore for `Alert` and can just `@derive` it.